package.json

```diff
@@ -90,6 +88,7 @@
   "peerDependencies": {
     "@babel/core": "*",
     "@babel/runtime": "*",
+    "@ledgerhq/hw-transport-webusb": "^5.51.0",
```

Comment:
Why did you make it a peer dependency? It's kinda weird that `hw-app-eth` is a common dependency, and this one is a peer one.